identify/README.md

```diff
@@ -0,0 +1,56 @@
+# Identify v1.0.0
+
+The identify protocol is used to identify a peer and its capabilities.
```

Comment:
can we unpack the definition of "identify a peer"? the recursive definition is confusing for a spec

Reply:
uhm ok; me loves recursion.

Reply:
i really can't think of a non-verbose way of expressing this.

Reply:
query, not interrogate.